### PR TITLE
The Messenger: add webhost auto connection steps to guide

### PR DESCRIPTION
**Pull request by alwaysintreble**
## What is this fixing or adding?
This new feature got added and The Messenger already supports it as part of the feature but i forgor to update docs.

### CODE REVIEW

File: worlds/messenger/docs/setup_en.md [hunk 1]
```diff
@@ -41,14 +41,25 @@ These steps can also be followed to launch the game and check for mod updates af
 
 ## Joining a MultiWorld Game
 
+### Automatic Connection on archipelago.gg
+
+1. Go to the room page for the game you are going to join.
+2. Click on your name on the left side.
```
Comment (ReverM) [inline worlds/messenger/docs/setup_en.md:47]: Tiny nitpick, but 
```suggestion
2. Click on your slot name on the left side.
```

File: worlds/messenger/docs/setup_en.md [hunk 2]
```diff
@@ -41,14 +41,25 @@ These steps can also be followed to launch the game and check for mod updates af
 
 ## Joining a MultiWorld Game
 
+### Automatic Connection on archipelago.gg
+
+1. Go to the room page for the game you are going to join.
```
Comment (ReverM) [inline worlds/messenger/docs/setup_en.md:46]:
```suggestion
1. Go to the room page of the MultiWorld you are going to join.
```
Game is used later and is confusing

File: worlds/messenger/docs/setup_en.md [hunk 3]
```diff
@@ -41,14 +41,25 @@ These steps can also be followed to launch the game and check for mod updates af
...
+2. Click on your name on the left side.
+3. Click the "The Messenger" button in the prompt.
+4. Follow the remaining prompts. This process will check that you have the mod installed and will also check for updates
+   before launching the game.
```
Comment (ReverM) [inline worlds/messenger/docs/setup_en.md:50]:
```suggestion
   before launching The Messenger.
```

File: worlds/messenger/docs/setup_en.md [hunk 4]
```diff
@@ -41,14 +41,25 @@ These steps can also be followed to launch the game and check for mod updates af
...
+5. Start a new save. You will already be connected in game and do not need to go through the menus.
```
Comment (ReverM) [inline worlds/messenger/docs/setup_en.md:51]:
```suggestion
5. Start a new save. You will already be connected in The Messenger and do not need to go through the menus.
```

**Review by ReverM — CHANGES_REQUESTED** [review]
The current steps uses the word game too much to refer to different things. It is slightly confusing

File: worlds/messenger/docs/setup_en.md [hunk 5]
```diff
@@ -41,14 +41,27 @@ These steps can also be followed to launch the game and check for mod updates af
...
 ## Joining a MultiWorld Game
 
+### Automatic Connection on archipelago.gg
+
+1. Go to the room page of the MultiWorld you are going to join.
+2. Click on your slot name on the left side.
+3. Click the "The Messenger" button in the prompt.
+4. Follow the remaining prompts. This process will check that you have the mod installed and will also check for updates
+   before launching The Messenger. If you are using the Steam version of The Messenger you may also get a prompt from
+   Steam asking if the game should be launched with arguments. These arguments are the URI with which the mod uses to
```
Comment (Exempt-Medic) [inline worlds/messenger/docs/setup_en.md:51]: I think you can just remove this word
```suggestion
   Steam asking if the game should be launched with arguments. These arguments are the URI which the mod uses to
```

**Review by Exempt-Medic — APPROVED** [review]
One small comment, the rest looks good to me

**Review by nicholassaylor — APPROVED** [review]
You might want to italicize "The Messenger" in the docs, but otherwise the info looks great!

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**